dump_frames.py

```diff
@@ -96,7 +97,8 @@ def dump_frames(video_path, output_directory, frames_per_second):
         if extract_all_frames:
             cmd = ['ffmpeg', '-i', video_path, name_format]
         else:
-            cmd = ['ffmpeg', '-i', video_path, '-vf', '"fps=%s"', name_format]
+            cmd = ['ffmpeg', '-i', video_path, '-vf', 
+                   'fps={}'.format(frames_per_second), name_format]
```

Comment:
d'oh! Looks like I added this bug recently when switching from moviepy to ffmpeg... Thanks!